src/components/SwapWidget/SwapWidget.tsx

```diff
@@ -863,7 +877,7 @@ const SwapWidget: FC<SwapWidgetPropsType> = ({
       <Overlay
         title={t("validatorErrors.unableSwap")}
         subTitle={t("validatorErrors.swapFail")}
-        onClose={async () => {
+        onCloseButtonClick={async () => {
```

Comment:
I would consider using "useCallback" for async buttons.

Reply:
Not written by me, but I see what I can do. :)